.github/workflows/obs-staging-shared.yml

```diff
@@ -91,6 +91,13 @@ jobs:
         run: osc service manualrun
         working-directory: ./${{ vars.OBS_PROJECT }}/${{ inputs.package_name }}
 
+      - name: Cleanup
+        # sometimes the "osc service" run does not cleanup properly all
+        # downloaded NPM package tarballs and they are accidentally added to the
+        # OBS package, so delete any TGZ files present
+        run: rm -f *.tgz
```

Comment:
how it will look in ci output? will it print all deleted files? maybe adding `rm -vf` would help to see what is removed to verify if something unexpected happen.